.github/workflows/build-test-images.yml

```diff
@@ -19,6 +19,35 @@ jobs:
           - extbuilder
           - exttester
           - failtester
+        pgversion:
+          - '13.9'
+          - '14.6'
+          - '15.1'
```

Comment:
I like the extra speed, however, I wonder if it will become a hassle having the versions in here.

I expect it to cause issues when we are bumping the postgres versions. Would there be a way that these can be discovered at runtime from the makefile? (I don't expect so, but would be worth an extra look).

What happens if we only bump the versions in the makefile and forget them here. I expect the actions to fail, but would be good to double check that.

Unless you have an other creative way of maintaining this dimension of the matrix.